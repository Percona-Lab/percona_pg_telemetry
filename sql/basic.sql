--
-- basic sanity
--

CREATE  EXTENSION percona_pg_telemetry;

-- Sleep so that waiting on agent becomes true
SELECT  pg_sleep(3);

SELECT  name
FROM    pg_settings
WHERE   name LIKE 'percona_pg_telemetry.%';

SELECT  percona_pg_telemetry_version();

SELECT  regexp_replace(regexp_replace(latest_output_filename, '\d{11,}', '<INSTANCE ID>', 'g'), '\d{6,}', '<TIMESTAMP>', 'g') AS latest_output_filename
        , pt_enabled
FROM    percona_pg_telemetry_status();

DROP    EXTENSION percona_pg_telemetry;
