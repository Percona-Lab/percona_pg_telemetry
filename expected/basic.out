--
-- basic sanity
--
CREATE  EXTENSION percona_telemetry;
-- Sleep so that waiting on agent becomes true
SELECT  pg_sleep(3);
 pg_sleep 
----------
 
(1 row)

SELECT  name
FROM    pg_settings
WHERE   name LIKE 'percona_telemetry.%';
                 name                  
-------------------------------------
 percona_telemetry.enabled
 percona_telemetry.files_to_keep
 percona_telemetry.path
 percona_telemetry.scrape_interval
(4 rows)

SELECT  percona_telemetry_version();
 percona_telemetry_version 
---------------------------
 1.0
(1 row)

SELECT  regexp_replace(regexp_replace(latest_output_filename, '\d{11,}', '<INSTANCE ID>', 'g'), '\d{6,}', '<TIMESTAMP>', 'g') AS latest_output_filename
        , pt_enabled
FROM    percona_telemetry_status();
               latest_output_filename               | pt_enabled 
----------------------------------------------------+------------
 ./percona_telemetry-<INSTANCE ID>-<TIMESTAMP>.json | t
(1 row)

DROP    EXTENSION percona_telemetry;
